src/redux/immutable/FetchableData.ts

```diff
@@ -19,7 +19,7 @@ const defaultValues: DefaultValues<Shared> = {
 };
 
 export class FetchableDataRecord<R> extends Record<Shared>(defaultValues) {
```

Comment:
I think there's a good case to be made to get rid of the whole fetchable data concept and it's not really relevant to us anymore

Reply:
Should I remove it in this PR or should I create a separate issue to do that?

Reply:
I need to test it properly / think about it harder :)

So open a separate issue for it :)